### PR TITLE
feat: #406 login signup

### PR DESCRIPTION
**Pull request by wjtwzgl12**

### CODE REVIEW

File: src/app/forgetPassword/forgetPassword.css
Comment (Yingxue0323) [inline src/app/forgetPassword/forgetPassword.css:1]: You should save all css files into styles folder instead of here

File: src/app/globals.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-
+@import 'bulma/css/bulma.css';
```
Comment (Yingxue0323) [inline src/app/globals.css:4]: Better to use customized bulma.scss from old code instead of importing the lib directly.

**Review by ucfyao — COMMENTED** [review]
1. During the refactoring process, keep the code clean by removing invalid comments. Maintain the English version.
2. During development, cultivate the habit of breaking down tasks. This can greatly avoid conflicts. Please resolve conflicts before submitting the PR.

**Comment by wjtwzgl12** [comment]
Done

**Review by ucfyao — COMMENTED** [review]
done

**Pull request merged**